### ISSUE TITLE
More file formats

### ISSUE THREAD

**Issue by StefanLobbenmeier** [mchome/flutter_advanced_networkimage#118]
I just discovered that this library can not render .tiff files. I had a bit of a look around the code, and I discovered that I might be able to use the Parameter postProcessing to process the data myself. I also found this library: https://github.com/brendan-duncan/image which might give me everything I need to process the tiff file. Is this the correct approach / has anyone done this before?

**Comment by mchome** [comment]
Yeah, if flutter engine(c++) does not support decoding tiff format, you can decode it in your approach(convert tiff to jpeg).

**Comment by amorenew** [comment]
@StefanLobbenmeier why you are use tiff?
